### ISSUE TITLE
install in blender 2.8 does not seem to work

### ISSUE THREAD

**Issue by marcovanderbijl** [addam/Export-Paper-Model-from-Blender#68]
First thanks for your work on this great addon!

The add-on did not seem to come included in my copy of blender 2.8 beta so I downloaded the zipfile and installed it, this seemed to work as blender reported it as such(see below). Unfortunately (even after restart) it did not appear in the /edit/preferences/add-ons/official, community or testing/ list so I could not select and use it.

install messages:
bpy.data.window_managers["WinMan"].addon_filter = 'Import-Export'
bpy.data.window_managers["WinMan"].addon_search = "paper model"
bpy.data.window_managers["WinMan"].addon_filter = 'All'
bpy.data.window_managers["WinMan"].addon_search = "paper model"
Modules Installed () from '/home/marco/Programs/blender-2.80-480a09a92f7f-linux-glibc224-x86_64/Export-Paper-Model-from-Blender-blender28.zip' into '/home/marco/.config/blender/2.80/scripts/addons'
bpy.data.window_managers["WinMan"].addon_search = "paper model"
bpy.data.window_managers["WinMan"].addon_filter = 'Import-Export'
bpy.data.window_managers["WinMan"].addon_search = ""
bpy.data.window_managers["WinMan"].addon_filter = 'User'
bpy.data.window_managers["WinMan"].addon_filter = 'Mesh'
bpy.data.window_managers["WinMan"].addon_filter = 'Development'
bpy.data.window_managers["WinMan"].addon_filter = 'All'

**Comment by addam** [comment]
Download and install this file (instead of the zip): https://raw.githubusercontent.com/addam/Export-Paper-Model-from-Blender/blender28/io_export_paper_model.py

By the way, what is "the zipfile" you downloaded?

**Comment by marcovanderbijl** [comment]
I selected branch:blender28 and then clicked the clone or download button, where in the popupbox one can select 'download ZIP'. I will try your direct link, but I suspect that is the same file I got earlier.

**Comment by addam** [comment]
aha, sorry, that's really the same file.
I think you're doing everything right. I'll try and reinstall Blender if I can reproduce the error.

**Comment by marcovanderbijl** [comment]
oh! I figured it out.
the 'addon install from file' installs it to .config/blender/... (where blender does not look for the file)
I found that the beta version actually did already have the script, but inside the blender install path itself.
I had to point /edit/preferences/file_paths/scripts to that folder (it was blank by default), then it was found! So I am now expecting that this issue is because of the beta status and will be automatically resolved once Blender makes 2.8 official.

However even though I replaced the existing (older version of the script) with your new one, it was marked with 'upgrade to 2.8 required'.
I could activate it and run it, though I have not tried actually exporting something.

I'll close the issue, sorry to have bothered you with this.
marco
